contracts/Staking.sol

```diff
@@ -567,7 +576,9 @@ contract Staking is
         uint256 proportion = (_talentAmount * MUL) / stake.talentAmount;
```

Comment:
start using math safe?

Reply:
yeah, absolutely! thanks for the reminder